src/Chapter14/Listing14.07.UsingthePlusEqualsAndMinusEqualsDelegateOperators.cs

```diff
@@ -16,18 +18,24 @@ public static void Main()
             Action<float> delegate2;
             Action<float>? delegate3;
 
-            // use Constructor syntax for C# 1.0
             delegate1 = heater.OnTemperatureChanged;
             delegate2 = cooler.OnTemperatureChanged;
 
             Console.WriteLine("Invoke both delegates:");
             delegate3 = delegate1;
+            #region HIGHLIGHT
             delegate3 += delegate2;
+            #endregion HIGHLIGHT
             delegate3(90);
 
             Console.WriteLine("Invoke only delegate2");
+            #region HIGHLIGHT
             delegate3 -= delegate1;
+            #endregion HIGHLIGHT
             delegate3!(30);
+            #region EXCLUDE
         }
     }
+    #endregion EXCLUDE
```

Comment:
Would it be simpler to just move the #region INCLUDE to just surround the inner section instead of the #exclude right next to an include?